export function conflictError() {
  return {
    name: "ConflictError",
    message: "Already exists!",
  };
}
